core/math/dynamic_bvh.h

```diff
@@ -182,7 +182,7 @@ class DynamicBVH {
 		Volume volume;
 		Node *parent = nullptr;
 		union {
-			Node *childs[2];
+			Node *childs[2]{};
```

Comment:
The constructor can be removed if this is a correct way to initialize both elements to `nullptr`.
```
		Node() {
			childs[0] = nullptr;
			childs[1] = nullptr;
		}
```